misk/src/main/kotlin/misk/web/jetty/JettyService.kt

```diff
@@ -425,3 +456,9 @@ private fun AbstractHTTP2ServerConnectionFactory.customize(webConfig: WebConfig)
     initialStreamRecvWindow = webConfig.jetty_initial_stream_recv_window
   }
 }
+
+private fun isJEP380Supported(path: String) : Boolean {
```

Comment:
this is awkward to test because `JavaVersion.VERSION` is a static assignment from the `java.version` system property. Setting/unsetting the version will affect other tests.

Reply:
updated the tests to assume running Java 16+ else ignore.

Reply:
Alternatively, you could make the Java version come from a separate, overridable method or pass it in as a default param so you can explicitly test each case.